src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,37 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static DateTimeFormatter formatter = DateTimeFormatter.ofPattern("d.MM.yyyy");
+    private LocalDate fromDate;
+    private LocalDate toDate;
+    private StringBuilder builder = new StringBuilder();
+
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int result = 0;
+        StringBuilder builder = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
```

Comment:
```suggestion
        StringBuilder reportBuilder = new StringBuilder("Report for period " + dateFrom + " - " + dateTo);
```
variable names should be informative